src/saving/SaveListItem.cs

```diff
@@ -189,6 +192,12 @@ public override void _GuiInput(InputEvent @event)
 
     public void LoadThisSave()
     {
+        if (type.Text == "Broken")
```

Comment:
I'm not happy with this solution, but I didn't find another solution.

Reply:
I think that the save shouldn't react to the invalid nature of the file. Instead loading *should* be attempted anyway.

Reply:
A save is only broken if there is an exception when untaring or when a file in the tar cannot be loaded. This means, that the save is really broken and will crash the game.

Reply:
Looking at the code, it seems possible that a save tar that is missing the info file will trigger an exception. However loading that save could work (I didn't look if the save loading also tries to read the info file, my memory says that it doesn't). So it's possible to have a broken save without the info but one that could load correctly.